Cargo.toml

```diff
@@ -71,6 +71,7 @@ forge-fmt = { version = "0.2.0", optional = true }
 # We don't use ethers-core directly, but need the correct version for the
 # build to work.
 ethers-core = { version = "2.0.10", optional = true }
+soroban-sdk = { version = "20.0.0-rc2", features = ["testutils"], optional = true }
```

Comment:
yes, and primarily it allows compiling contracts natively -- as rust code that registers itself as a trait object with the soroban host, rather than as a chunk of wasm VM code.